### PR TITLE
Application Commands

### PR DESCRIPTION
**Pull request by Seniatical**
Support for application commands such as slash commands

### CODE REVIEW

**Comment by Daftscientist** [comment]
Slash commands.

**Comment by MrPotato374** [comment]
nice

File: examples/application_commands/bookmark.py [hunk 1]
```diff
@@ -43,4 +43,4 @@ async def on_error(self, interaction: Interaction, exc_info: tuple) -> Any:
 client.register_application_command(Bookmark(client=client))
 
 if __name__ == "__main__":
-    client.run("NzQxNzE0NTcxNjYzNzY5NjUw.Xy7lhg.r-QKEtP6pA_JQ0xzhtTSk3006dU")
```
Comment (Seniatical) [inline examples/application_commands/bookmark.py]: dipstick

Comment (Seniatical) [reply examples/application_commands/bookmark.py]: regenerated the bloody token

**Comment by Seniatical** [comment]
Its ready!

**Pull request merged**